README.md

```diff
@@ -32,8 +87,4 @@ zaporter:sds011:v1-fake
 }
 ```
 
-# Building
-`make build` -> `bin/module`
-
-# Linting
-`make lint`
+You can view sensor readings on [your machine's **Control** tab in the Viam app](https://app.viam.com/) or by using the [sensor API](https://docs.viam.com/components/sensor).
```

Comment:
```suggestion
You can view sensor readings on [your machine's Control tab in the Viam app](https://app.viam.com/) or by using the [sensor API](https://docs.viam.com/components/sensor).
```

Bold inside the link seems a bit hard to read

Reply:
I hear ya, but this is just how we do it throughout the docs to indicate that it's a clickable UI element. Will leave this one as-is